docs/Release Notes/release-notes.md

```diff
@@ -6,7 +6,88 @@ slug: /release-notes/2022
 
 ---
 
-## v0.1.4
+## v0.1.9
+
+**November 25, 2022**
+
+**paralus/paralus**
+
+**Added**
+
+- Added database auditlog storage option [niravparikh05](https://github.com/niravparikh05)
+
+Find more details in [paralus/paralus](https://github.com/paralus/paralus).
+
+---
+
+## v0.1.8
+
+**November 04, 2022**
+
+**paralus/paralus**
+
+**Added**
+
+- Added last login field to user spec from [akshay196](https://github.com/akshay196)
+
+Find more details in [paralus/paralus](https://github.com/paralus/paralus).
+​
+**paralus/dashboard**
+
+**Added**
+
+- Show last access time in users list page from [akshay196](https://github.com/akshay196)
+
+**Fixed**
+
+- Fix for IdP users unable to login from [niravparikh05](https://github.com/niravparikh05)
+- Fix wrong cluster delete success message for non-priviledged users from [niravparikh05](https://github.com/niravparikh05)
+
+Find more details in [paralus/dashboard](https://github.com/paralus/dashboard).
+
+---
+​
+## v0.1.7
+
+**October 14, 2022**
+
+​
+**paralus/paralus**
+
+**Fixed**
+
+- Fixed creating project scoped role failed from cli [niravparikh05](https://github.com/niravparikh05)
+
+Find more details in [paralus/paralus](https://github.com/paralus/paralus).
```

Comment:
this and below line comes up in the same line in preview

Reply:
Fixed.